config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
> So far it is only done in the dummy app and I think that is enough
You just said that...

You want to run this thing on production? The thing that `eval`s code from a YAML file? Seriously?

I think it's best to keep the actual codebase as small as possible and only run this in dev mode. 

One more thing: While starting the dummy app from the target app works for now, it won't work with the released gem: The `spec` directory is usually not packaged into the `.gem` files. So we either have to do that or come up with a better solution.